Code/MinimalLib/README.md

```diff
@@ -85,60 +92,26 @@ To use RDKit, load the javascript file and instantiate the wasm module inside th
 
 ```
 
-## Usage
-
-See the getting started demo at https://unpkg.com/@rdkit/rdkit/Code/MinimalLib/dist/GettingStartedInJS.html .
-
-Follow the examples of this page to see the various ways to use the JavaScript release of RDKit.
-
 ## Live demos
 
-- From this npm package: https://unpkg.com/@rdkit/rdkit/Code/MinimalLib/dist/GettingStartedInJS.html
-- From this npm package: https://unpkg.com/@rdkit/rdkit/Code/MinimalLib/dist/demo.html
+If you are using the MinimalLib for the first time, see the getting started examples at https://www.rdkitjs.com/ .
 
-## Contributing
+### All live demos
 
-### Preparing a new release of the package
+- RDKit.js website: https://www.rdkitjs.com/
+- RDKit.js usage with React.js: https://react.rdkitjs.com/
+- Legacy examples #1: https://unpkg.com/@rdkit/rdkit/dist/GettingStartedInJS.html
+- Legacy examples #2: https://unpkg.com/@rdkit/rdkit/dist/demo.html
 
-Make sure you are at the root of the [RDKit](https://github.com/rdkit/rdkit) GitHub project, and on the branch and version of the project you want to release. **Note that no commits should occur during the release process.**
+## Building the MinimalLib
 
-#### Step 1: Set the release version in package.json
+Make sure you are at the root of the [MinimalLib](https://github.com/rdkit/rdkit/tree/master/Code/MinimalLib), and run the following script.
 
 ```bash
-npm --no-git-tag-version version <semver version matching an RDKit release>
-# Example npm --no-git-tag-version version 2021.3.1
+bash scripts/build_rdkitjs.sh <RDKit git release tag name>
```

Comment:
Does this script exist?

Reply:
yep, see here https://github.com/rdkit/rdkit/blob/master/Code/MinimalLib/scripts/build_rdkitjs.sh